js/stringhelper.js

```diff
@@ -0,0 +1,85 @@
+/**helper functions*/
+YoastSEO_StringHelper = function() {};
+/**
+ * removes strings from array and replaces them with keyword.
+ * @param textString
+ * @param stringsToRemove []
+ * @param replacement (default == space)
+ * @returns {textString}
+ */
+YoastSEO_StringHelper.prototype.replaceString = function( textString, stringsToRemove, replacement ){
+    if( typeof replacement === "undefined" ){
+        replacement = " ";
+    }
+    textString = textString.replace( this.stringToRegex(stringsToRemove), replacement );
+    return this.stripSpaces( textString );
+};
+
+/**
+ * matches string with given array of strings to match.
+ * @param textString
+ * @param stringsToMatch
+ * @returns {matches}
+ */
+YoastSEO_StringHelper.prototype.matchString = function( textString, stringsToMatch ){
+    return textString.match( this.stringToRegex( stringsToMatch, false ) );
+};
+
+/**
+ * matches string with regex, returns the number of matches
+ * @param textString
+ * @param regex
+ * @returns {number}
+ */
+YoastSEO_StringHelper.prototype.countMatches = function( textString, regex ){
+    var count = 0;
+    var matches = textString.match( regex );
+    if( matches !== null ){
+        count = matches.length;
```

Comment:
Why don't you return matches.length immediately? That way you won't need to define count, as you can return 0 if the if statement does not match.